src/System.Private.ServiceModel/src/Internals/WcfEventSource.cs

```diff
@@ -2926,6 +2962,9 @@ public class Keywords
             public const EventKeywords WFActivities = (EventKeywords)0x2000000;
             public const EventKeywords WFServices = (EventKeywords)0x4000000;
             public const EventKeywords WFInstanceStore = (EventKeywords)0x8000000;
+            // This DummyKeyword is used to emit events that aren't being listened for in
+            // order to initialize EventSource with the correct values for the eventlog channels
+            public const EventKeywords DummyKeyword = (EventKeywords) 0x080000000;
```

Comment:
I am surprised you need this new dummy keyword.  I would expect just leaving the Keywords property off the 4 events above would work just fine.